solutions/community-deploy/cloud/AWS/README.md

```diff
@@ -48,72 +45,97 @@ os_version          = "CentOS 7.9.2009 x86_64"
 region              = "ap-south-1"
 security_group_cidr = "134.204.222.36/32"
 key_name            = "devops-key"
-
+instance_count      = "3"
+ebs_volume_count    = "9"
+ebs_volume_size     = "10"
+tag_name            = "cortx-multinode"
 ```
 
-## Create AWS instance
-
-- Execute Terraform code (as shown below) to create AWS instance for CORTX Build and Deployment.  
-- The command will display public-ip on completion. Use this public-ip to connect AWS instance using SSH Protocol. 
+### Execute Instructions to create AWS Instances and Network & Storage Configuration
+- Execute Terraform code (as shown below) to create AWS instances for CORTX Build and Deployment.
 ```
 terraform validate && terraform apply -var-file user.tfvars --auto-approve
 ```
+- Execute the following commands to find the Public and Private ip addresses from local host by setting as environment variable.
 
-## Network and Storage Configuration.
-
-- Execute `/home/centos/setup.sh` to setup Network and Storage devices for CORTX. Script will reboot instance on completion. 
+**Public ip address:**
+```
+export PUBLIC_IP=`terraform show -json terraform.tfstate | jq .values.outputs.aws_instance_public_ip_addr.value 2>&1 | tee ip_public.txt  | tr -d '",[]' | sed '/^$/d'`
```

Comment:
`| tr -d '",[]' | sed '/^$/d'` is used in both at line 63 and 64. Can we remove any one of it

Reply:
fixed